endpoint/response.go

```diff
@@ -75,20 +83,29 @@ func EncodePhonebook(data []*model.Phonebook) []*Phonebook {
 		}
 
 		encodeData := &Phonebook{
-			ID:           v.ID,
-			PhoneNumbers: phoneNumbers,
-			Description:  v.Description,
-			Name:         v.Name,
-			Address:      v.Address,
-			Latitude:     v.Latitude,
-			Longitude:    v.Longitude,
-			Status:       v.Status,
-			Category:     v.Category,
-			Distance:     v.Distance,
+			ID:            v.ID,
+			PhoneNumbers:  phoneNumbers,
+			Description:   v.Description,
+			Name:          v.Name,
+			Address:       v.Address,
+			CoverImageURL: fmt.Sprintf("%s/%s", cfg.AppStoragePublicURL, v.CoverImageURL),
+			Latitude:      v.Latitude,
+			Longitude:     v.Longitude,
+			Status:        v.Status,
+			StatusLabel:   GetStatusLabel[v.Status]["ina"],
```

Comment:
@setiadijoe Some leftovers :grin: 
```suggestion
			StatusLabel:   GetStatusLabel[v.Status]["id"],
```